2021-05-02 09:10:39,270 INFO    MainThread:71227 [wandb_setup.py:_flush():69] setting env: {}
2021-05-02 09:10:39,270 INFO    MainThread:71227 [wandb_setup.py:_flush():69] setting login settings: {}
2021-05-02 09:10:39,270 INFO    MainThread:71227 [wandb_init.py:_log_setup():336] Logging user logs to /Users/merden/Research/denoiser/wandb/run-20210502_091039-e0a5sesa/logs/debug.log
2021-05-02 09:10:39,271 INFO    MainThread:71227 [wandb_init.py:_log_setup():337] Logging internal logs to /Users/merden/Research/denoiser/wandb/run-20210502_091039-e0a5sesa/logs/debug-internal.log
2021-05-02 09:10:39,271 INFO    MainThread:71227 [wandb_init.py:init():369] calling init triggers
2021-05-02 09:10:39,271 INFO    MainThread:71227 [wandb_init.py:init():374] wandb.init called with sweep_config: {}
config: {}
2021-05-02 09:10:39,271 INFO    MainThread:71227 [wandb_init.py:init():418] starting backend
2021-05-02 09:10:39,271 INFO    MainThread:71227 [backend.py:_multiprocessing_setup():69] multiprocessing start_methods=spawn,fork,forkserver, using: spawn
2021-05-02 09:10:39,282 INFO    MainThread:71227 [backend.py:ensure_launched():123] starting backend process...
2021-05-02 09:10:39,287 INFO    MainThread:71227 [backend.py:ensure_launched():127] started backend process with pid: 71234
2021-05-02 09:10:39,289 INFO    MainThread:71227 [wandb_init.py:init():423] backend started and connected
2021-05-02 09:10:39,291 INFO    MainThread:71227 [wandb_init.py:init():465] updated telemetry
2021-05-02 09:10:39,292 INFO    MainThread:71227 [wandb_init.py:init():484] communicating current version
2021-05-02 09:10:40,318 INFO    MainThread:71227 [wandb_init.py:init():489] got version response upgrade_message: "wandb version 0.10.28 is available!  To upgrade, please run:\n $ pip install wandb --upgrade"

2021-05-02 09:10:40,319 INFO    MainThread:71227 [wandb_init.py:init():497] communicating run to backend with 30 second timeout
2021-05-02 09:10:40,486 INFO    MainThread:71227 [wandb_init.py:init():522] starting run threads in backend
2021-05-02 09:10:42,131 INFO    MainThread:71227 [wandb_run.py:_console_start():1524] atexit reg
2021-05-02 09:10:42,132 INFO    MainThread:71227 [wandb_run.py:_redirect():1371] redirect: SettingsConsole.REDIRECT
2021-05-02 09:10:42,133 INFO    MainThread:71227 [wandb_run.py:_redirect():1376] Redirecting console.
2021-05-02 09:10:42,135 INFO    MainThread:71227 [wandb_run.py:_redirect():1438] Redirects installed.
2021-05-02 09:10:42,135 INFO    MainThread:71227 [wandb_init.py:init():546] run started, returning control to user process
2021-05-02 09:10:42,151 INFO    MainThread:71227 [wandb_watch.py:watch():39] Watching
2021-05-02 09:10:42,152 INFO    MainThread:71227 [wandb_watch.py:watch():39] Watching
2021-05-02 09:10:42,152 INFO    MainThread:71227 [wandb_watch.py:watch():39] Watching
2021-05-02 09:10:42,439 INFO    MainThread:71227 [wandb_run.py:_atexit_cleanup():1494] got exitcode: 1
2021-05-02 09:10:42,440 INFO    MainThread:71227 [wandb_run.py:_restore():1466] restore
2021-05-02 09:10:44,598 INFO    MainThread:71227 [wandb_run.py:_wait_for_finish():1616] got exit ret: file_counts {
  wandb_count: 1
}
pusher_stats {
  uploaded_bytes: 658
  total_bytes: 658
}

2021-05-02 09:10:45,522 INFO    MainThread:71227 [wandb_run.py:_wait_for_finish():1616] got exit ret: file_counts {
  wandb_count: 6
}
pusher_stats {
  uploaded_bytes: 658
  total_bytes: 2533
}

2021-05-02 09:10:45,658 INFO    MainThread:71227 [wandb_run.py:_wait_for_finish():1616] got exit ret: file_counts {
  wandb_count: 6
}
pusher_stats {
  uploaded_bytes: 658
  total_bytes: 2533
}

2021-05-02 09:10:45,783 INFO    MainThread:71227 [wandb_run.py:_wait_for_finish():1616] got exit ret: file_counts {
  wandb_count: 6
}
pusher_stats {
  uploaded_bytes: 2533
  total_bytes: 2533
}

2021-05-02 09:10:45,885 INFO    MainThread:71227 [wandb_run.py:_wait_for_finish():1616] got exit ret: file_counts {
  wandb_count: 6
}
pusher_stats {
  uploaded_bytes: 2533
  total_bytes: 2533
}

2021-05-02 09:10:45,988 INFO    MainThread:71227 [wandb_run.py:_wait_for_finish():1616] got exit ret: file_counts {
  wandb_count: 6
}
pusher_stats {
  uploaded_bytes: 2533
  total_bytes: 2533
}

2021-05-02 09:10:46,093 INFO    MainThread:71227 [wandb_run.py:_wait_for_finish():1616] got exit ret: file_counts {
  wandb_count: 6
}
pusher_stats {
  uploaded_bytes: 2533
  total_bytes: 2533
}

2021-05-02 09:10:46,199 INFO    MainThread:71227 [wandb_run.py:_wait_for_finish():1616] got exit ret: file_counts {
  wandb_count: 6
}
pusher_stats {
  uploaded_bytes: 2533
  total_bytes: 2533
}

2021-05-02 09:10:46,305 INFO    MainThread:71227 [wandb_run.py:_wait_for_finish():1616] got exit ret: done: true
exit_result {
}
file_counts {
  wandb_count: 6
}
pusher_stats {
  uploaded_bytes: 2533
  total_bytes: 2533
}

2021-05-02 09:10:47,402 INFO    MainThread:71227 [wandb_run.py:_show_files():1838] logging synced files
